.github/workflows/build.yml

```diff
@@ -54,19 +54,27 @@ jobs:
           git config --global core.eol lf
           git config --global core.symlinks true
       - uses: actions/checkout@v2
-      - name: Derive pack version from branch name
+      - name: Derive pack version from branch name Unix
```

Comment:
On lifecycle we wrote a helper binary to do this: https://github.com/buildpacks/lifecycle/blob/11ca8ef99c1626fa34ea087e74993d8746c7cf61/tools/version/main.go#L13-L24 to avoid the cross-os headache. IDK if it's worth doing here though since you already have something working.

Reply:
Ohhh this is cool 👍